To-do list/Controllers/Cell/TaskTableViewCell.swift

```diff
@@ -28,8 +28,8 @@ class TaskTableViewCell: UITableViewCell {
     
     func configure(_ task: Task) {
         titleLabel.text = task.title
-        descriptionLabel.text = task.description
-        deadlineLabel.text = task.deadline.formatDate()
+        contentLabel.text = task.content
+        deadlineLabel.text = String(task.deadline.dropLast(3))
```

Comment:
don't modify model's value with `.dropLast(3)`. If ever date format changed in the response - your approach will fail. Please use DateFormatter and convert date string properly to desired new format

Reply:
done